core/src/core-api/config.js

```diff
@@ -286,7 +286,7 @@ class LuigiConfig {
    * @memberof Configuration
    * @example
    * Luigi.clearNavigationCache();
-   * @since 1.19.0
+   * @since NEXT_RELEASE
```

Comment:
this and the related docu below [luigi-client-api.md](https://github.com/SAP/luigi/pull/2715/files#diff-4ff8ced4162e9cb3719d7544de4ec0ff185efbc3d3d655b610c857bb224e5f26), shouldn't be in this PR as well